index.js

```diff
@@ -100,7 +97,14 @@ class Plugin extends PluginMiniAccounts {
         } else if (response[0] === IlpPacket.Type.TYPE_ILP_FULFILL) {
           // TODO: should await, or no?
           const { amount } = IlpPacket.deserializeIlpPrepare(ilp)
-          if (amount !== '0' && this._moneyHandler) this._moneyHandler(amount)
+
+          if (!this._moneyHandler) {
+            throw new Error('no money handler registered')
+          }
+
+          if (amount !== '0') {
+            this._moneyHandler(amount)
```

Comment:
it's more efficient to just configure the connector not to track this plugin's balance than it is to pretend a settlement came in on each packet

Reply:
Could you elaborate a little more on what you mean by pretending a settlement came in? I don't have a full understanding of how the connector's balance logic interacts with this

Reply:
if you call `moneyHandler`, then the connector will increase their balance for this plugin. When they send a packet out on this plugin, they'll decrease the balance. But here, `moneyHandler` is just being called after a fulfill, not after a settlement. It means that for every packet, the balance gets reset to zero. To achieve the same effect, you can just configure the connector to not track a balance at all for this plugin